README.md

```diff
@@ -37,7 +37,7 @@ Archived code base corresponding to publication: https://zenodo.org/record/74199
     - The code was tested with the 2017, 2019, and 2022 Community editions.
 2. **Linux only**: Install OpenBLAS libraries
     - `sudo apt-get install libopenblas-base`
-
+3. **MacOs**: As of now this workflow is not supported on Mac computers with Apple Silicon. May work on a Mac with intel processor.

```

Comment:
@aymanhab do I understand well that it does work in some cases? you wrote this in the other PR: "With that change example_kinetics.py works on apple silicon using x86_64 architecture/stack"

Reply:
After the change in the other PR #138 yes, it works on x86_64 stack. I'll update accordingly though users who do not clone the repo fresh will not see the change. Ideally changes get grouped into a tag so users can tell if they need to update rather than chase bleeding edge changes but we're not there yet.